### PR TITLE
Wrapper for pipelign version 0.2

### PR DESCRIPTION
**Pull request by davebx**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection

### CODE REVIEW

File: tools/pipelign/pipelign.xml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        <param argument="--simPer" type="float" value="0.8" min="0" max="1" label="Percent sequence similarity for clustering" help="Default: 0.8" />
+        <param argument="--run" type="select" label="Parallelization version">
+            <option value="J">Joblib</option>
+            <option value="G">GNU</option>
+        </param>
+        <param argument="--merge" type="select" label="Merge strategy">
+            <option value="P">Parallel</option>
+            <option value="C">Consensus</option>
+        </param>
+        <param argument="--mIterateLong" type="integer" value="1" label="Number of iterations to refine long alignments" help="Default: 1" />
```
Comment (bgruening) [inline tools/pipelign/pipelign.xml]: min=0 or 1?

File: tools/pipelign/pipelign.xml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        <param argument="--mIterateMerge" type="integer" value="1" label="Number of iterations to refine merged alignment" help="Default: 1" />
```
Comment (bgruening) [inline tools/pipelign/pipelign.xml]: likewise

File: tools/pipelign/pipelign.xml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+<tool id="pipelign" name="Automated multiple sequence" version="@VERSION@+galaxy0">
+    <description>alignment with pipelign</description>
+    <macros>
+        <token name="@VERSION@">0.2</token>
+    </macros>
+    <requirements>
+        <requirement type="package" version="@VERSION@">pipelign</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        pipelign -d files -c -i '$inFile' -o '$outFile' -t $lenThr -a $alphabet
```
Comment (bgruening) [inline tools/pipelign/pipelign.xml]: can you add linebreaks here

File: tools/pipelign/pipelign.xml [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+        <param argument="--lenThr" type="float" value="0.7" min="0" max="1" label="Length threshold for full sequences" help="Default: 0.7" />
+        <param argument="--alphabet" type="select" label="Alphabet for input sequences">
+            <option value="dna">DNA</option>
+            <option value="rna">RNA</option>
+            <option value="aa">Amino Acids</option>
+        </param>
+        <param argument="--keepOrphans" type="boolean" truevalue="--keepOrphans" falsevalue="" label="Add fragments without clusters" />
+        <param argument="--keepBadSeqs" type="boolean" truevalue="--keepBadSeqs" falsevalue="" label="Add long sequences with too many ambiguous residues" />
+        <param argument="--simPer" type="float" value="0.8" min="0" max="1" label="Percent sequence similarity for clustering" help="Default: 0.8" />
+        <param argument="--run" type="select" label="Parallelization version">
```
Comment (bgruening) [inline tools/pipelign/pipelign.xml]: Is that really a parameter that we need to expose to the user? Does it matter?

File: tools/pipelign/pipelign.xml (hunk 3 above)
Comment (davebx) [reply tools/pipelign/pipelign.xml]: I think it's a lot more readable now

File: tools/pipelign/pipelign.xml [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+        <param argument="--ambigPer" type="float" value="0.1" min="0" max="1" label="Percent sequence similarity for clustering" />
+        <param argument="--excludeClusters" type="boolean" truevalue="--excludeClusters" falsevalue="" label="Exclude clusters from final alignment" />
+    </inputs>
+    <outputs>
+        <data name="outFile" format="fasta" />
+    </outputs>
+    <tests>
+        <test>
+            <param name="inFile" value="pipelign-in1.fasta" />
+            <output name="outFile" file="pipelign-out1.fasta" compare="sim_size" />
```
Comment (bgruening) [inline tools/pipelign/pipelign.xml]: @davebx maybe a fancy assert would be better, I'm not sure how differentiation you expect.

**Comment by davebx** [comment]
Looks like the test is failing because asmmhossain/pipelign#4 didn't make it into the bioconda recipe. I'm fixing that now.

**Pull request merged**